pg_search/src/postgres/storage/metadata.rs

```diff
@@ -59,6 +59,9 @@ pub struct MetaPageData {
 
     /// The block where our FSM starts
     fsm: pg_sys::BlockNumber,
+
+    /// The header block for a [`LinkedItemsList<SegmentMergeEntry>]`
+    segment_meta_garbage: pg_sys::BlockNumber,
```

Comment:
Is this the same ordering of this struct as on enterprise? Does that matter...?

Reply:
Yes, it's the same ordering and it does indeed matter.